src/sasquatchbackpack/fastapi.py

```diff
@@ -1,44 +1,47 @@
 """Backpack FastAPI module."""
 
 from fastapi import FastAPI
+from pydantic import BaseModel
 
 from sasquatchbackpack.sources.usgs.commands import usgs_earthquake_data
 
 app = FastAPI()
 
 
-@app.get(
-    "/sources/usgs/earthquake_data/{days}/{hours}/{radius}/{latitude}/{longitude}/{lower}/{upper}/{publish}/{method}"
-)
-async def read_item(
-    days: int,
-    hours: int,
-    radius: int,
-    latitude: int,
-    longitude: int,
-    lower: int,
-    upper: int,
-    publish: bool,  # noqa: FBT001
-    method: str,
-) -> None:
+class EarthquakeParams(BaseModel):
+    """{days}/{hours}/{radius}/{latitude}/{longitude}/{lower}/{upper}/{publish}/{method}."""
+
+    days: int
+    hours: int
+    radius: int
+    latitude: int
+    longitude: int
+    lower: int
+    upper: int
+    publish: bool
+    method: str
+
+
+@app.post("/sources/usgs/earthquake_data/")
```

Comment:
A small suggestion here, it's best practice to use hyphens (-) instead of underscores (_) to separate words in URLs. 
Since `-data` doesn't add much consider just `/sources/usgs/earthquake`